eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -321,89 +319,69 @@ jobs:
-
-              if (Test-Path $changeLogPath) {
-                Copy-Item sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}/CHANGELOG.md $outputPath
-              } else {
-                Write-Warning "$changeLogPath does not exist"
-              }
-            displayName: Copy CHANGELOG.md to package artifact
-
-      # This assumes that there are entries in parameters.Artifacts
-      - task: PublishPipelineArtifact@1
-        inputs:
-          artifactName: packages
-          path: $(Build.ArtifactStagingDirectory)/packages
-
-      # Generate Documentation
-      - pwsh: |
-          $setupFile="doxygen-1.8.18-setup.exe";
-
-          Invoke-WebRequest `
-            -MaximumRetryCount 10 `
-            -Uri "https://azuresdktooling.blob.core.windows.net/dependencies/$setupFile" `
-            -OutFile $setupFile `
-          | Wait-Process;
-
-          Start-Process -Filepath .\$setupFile -ArgumentList @("/VERYSILENT") -Wait
-        workingDirectory: $(Agent.TempDirectory)
-        displayName: Download and Install Doxygen
-
-      - ${{ each artifact in parameters.Artifacts }}:
-        - pwsh: Write-Host "##vso[task.setvariable variable=PackageVersion]$(Get-Content .\sdk\${{ parameters.ServiceDirectory }}\${{ artifact.Path }}\version.txt)"
-          displayName: Set $(PackageVersion) from version.txt file - ${{ artifact.Name }}
-        - pwsh: >-
-            python eng/docs/api/generate_docs.py
-            --TemplateFile="eng/docs/api/Doxyfile.template"
-            --PackagePath="sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}"
-            --PackageName="${{ artifact.Name }}"
-            --PackageVersion="$(PackageVersion)"
-            --AssetsPath="eng/docs/api/assets"
-            --DoxygenPath="c:\\program files\\doxygen\\bin\\doxygen.exe"
-          displayName: Generate docs - ${{ artifact.Name }}
-
-        - pwsh: New-Item -ItemType directory -Path $(Build.ArtifactStagingDirectory) -Name docs/${{ artifact.Name }}
-          displayName: Create doc artifact location - ${{ artifact.Name }}
-
-        - task: CopyFiles@2
-          inputs:
-            sourceFolder: sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}/html
-            targetFolder: $(Build.ArtifactStagingDirectory)/docs/${{ artifact.Name }}
-          displayName: Copy documentation - ${{ artifact.Name }}
-
-        - pwsh: |
-            $packageInfoPath = Join-Path -Path $(Build.ArtifactStagingDirectory) packages/${{ artifact.Name }}/package-info.json
-            Copy-Item -Path $packageInfoPath -Destination $(Build.ArtifactStagingDirectory)/docs/${{ artifact.Name }}/package-info.json
-          displayName: Copy package-info.json to documentation path
-
-      - task: PublishPipelineArtifact@1
-        inputs:
-          artifactName: docs
-          path: $(Build.ArtifactStagingDirectory)/docs
+
+- job: GenerateReleaseArtifacts
```

Comment:
I assume this isn't part of the release template because we want to do this for every build?

Reply:
Given that doc generation is now a matter of running `doxygen Doxyfile` in the root of the repo we can exclude this except for manual builds.